### DISCUSSION TITLE
VNote的表格问题

### DISCUSSION THREAD

**Discussion by Lishengtai12138** [tamlok/vnote#2419] [Q&A] [answered]
![image](https://github.com/vnotex/vnote/assets/114505641/c464f6c7-f39d-4545-83bf-049c27fbd580)
如图，使用表格语法后，在阅读模式没有显示表格，但我在markdowm官网使用该语法时却能正常使用
![image](https://github.com/vnotex/vnote/assets/114505641/29274145-1be2-4700-854f-46139672eb7a)

**Comment by tamlok** [comment, marked as answer]
第9行前面加一个空行

**Reply by Lishengtai12138** [reply]
谢谢大佬
